odoo/addons/base/models/ir_qweb.py

```diff
@@ -146,138 +153,50 @@ def _directives_eval_order(self):
 
     # compile directives
 
-    def _compile_directive_lang(self, el, options):
-        lang = el.attrib.pop('t-lang', get_lang(self.env).code)
-        if el.get('t-call-options'):
-            el.set('t-call-options', el.get('t-call-options')[0:-1] + u', "lang": %s}' % lang)
-        else:
-            el.set('t-call-options', u'{"lang": %s}' % lang)
-        return self._compile_node(el, options)
+    def _compile_directive_lang(self, el, options, indent):
+        lang = el.attrib.pop('t-lang', "'%s'" % get_lang(self.env).code)
+        el.attrib['t-options-lang'] = lang
+        return self._compile_node(el, options, indent)
 
-    def _compile_directive_call_assets(self, el, options):
+    def _compile_directive_call_assets(self, el, options, indent):
         """ This special 't-call' tag can be used in order to aggregate/minify javascript and css assets"""
         if len(el):
             raise SyntaxError("t-call-assets cannot contain children nodes")
 
-        # nodes = self._get_asset_nodes(bundle, options, css=css, js=js, debug=values.get('debug'), async=async, values=values)
```

Comment:
Remaining commented code to remove.

Reply:
It was removed. Do you want us to keep it?